.github/workflows/test-pr-command.yml

```
@@ -0,0 +1,163 @@
+    - name: Run Pytest (Flaky Only)
+      continue-on-error: true
+      timeout-minutes: 60
+      env:
+        GCP_GSM_CREDENTIALS: ${{ secrets.GCP_GSM_CREDENTIALS }}
+      run: >
+        poetry run pytest
+        --verbose
+        -m "flaky and not super_slow"
+
+    - name: Post CI Success to GitHub
+      run: |
```

actionlint

shellcheck reported issue in this script: SC2046:warning:2:70: Quote this to prevent word splitting

(shellcheck)